packages/workers-shared/router-worker/src/worker.ts

```diff
@@ -55,7 +79,10 @@ export default {
 			);
 
 			const hasStaticRouting = env.CONFIG.static_routing !== undefined;
-			const config = applyConfigurationDefaults(env.CONFIG);
+			const [config, eyeballConfig] = applyConfigurationDefaults(
+				env.CONFIG,
+				env.EYEBALL_CONFIG
+			);
```

Comment:
Personally I'd rather just have two functions here since they are completely independent of each other:

```suggestion
			const config = applyConfigurationDefaults(env.CONFIG);
			const eyeballConfig = applyEyeballConfigurationDefaults(env.EYEBALL_CONFIG);
```